tdrs-backend/manifest.buildpack.yml

```diff
@@ -1,7 +1,7 @@
 version: 1
 applications:
 - name: tdp-backend
-  memory: 2G
+  memory: 1G
```

Comment:
we should keep an eye on this once this merges to develop, can someone please test submitting large files to ensure we are passed the network error that was resolved in #2354 cc: @andrew-jameson @lilybl1